test/runtests.jl

```diff
@@ -8,9 +8,13 @@ using SpecialFunctions: AmosException, f64
 
 # useful test functions for relative error, which differ from isapprox
 # in that relerrc separately looks at the real and imaginary parts
```

Comment:
Oh, I see, you merged `relerr` and `relerrc` — that's fine, then.  But please change the comment as well.